indigo_drivers/mount_synscan/indigo_mount_synscan_driver.c

```diff
@@ -172,28 +174,28 @@ bool synscan_configure(indigo_device* device) {
 	if (!PRIVATE_DATA->mountConfigured) {
 		//  Query axis 360 degree rotation steps
 		if (!synscan_total_axis_steps(device, kAxisRA, &PRIVATE_DATA->raTotalSteps))
-			return false;
+			INDIGO_DRIVER_ERROR(DRIVER_NAME, "Unable to fetch RA axis steps per rotation");
```

Comment:
Is there any specific reason to continue but not return failure?

Reply:
Short version: it makes this mount work. 

I didn't see that this value was being used in any significant way, and by patching out all the failing queries it allows the mount to connect.

Reply:
Excuse me does it work or just connects? I mean goto, park etc... Does it behave normally?
My concern is that if it replies in a not expected way to dose commands it may well reply differently to some other commands which is not detected by the driver and to behave erratically. I would rather investigate a bit the messages and try to figure out why is that. But if you confirm it really works with all the features I will merge the PR.